src/pages/questionPage.js

```diff
@@ -6,31 +6,45 @@ import {
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
+import { changeProgress } from '../views/progressBar.js';
+
 
 export const initQuestionPage = () => {
   const userInterface = document.getElementById(USER_INTERFACE_ID);
   userInterface.innerHTML = '';
 
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
 
-  const questionElement = createQuestionElement(currentQuestion.text);
+  const questionElement = createQuestionElement(currentQuestion.text, quizData.currentQuestionIndex + 1);
 
   userInterface.appendChild(questionElement);
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+
   }
 
+// this code disabled and active button
+  document
+  .getElementById(NEXT_QUESTION_BUTTON_ID)
+  .setAttribute('disabled', true)
+
+  // Add event listener to the button
   document
-    .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .addEventListener('click', nextQuestion);
+  .getElementById(NEXT_QUESTION_BUTTON_ID)
+  .removeAttribute('disabled')
+
+  document
+  .getElementById(NEXT_QUESTION_BUTTON_ID)
+  .addEventListener('click', nextQuestion);
+
+  const currentBar = changeProgress((quizData.currentQuestionIndex * 10)+10); // Progressbar line
```

Comment:
Here is an instance where the number `10` should not be hard coded in.

Reply:
Why I added 10 because when i start the new game I want to do first step, 10 - it is default number.

Reply:
progressbar.style.width = `${progress}%` - this number changed this line, so I think it is not really important

Reply:
Oh gotcha! My mistake.